src/components/Button/__tests__/button-snapshot-tests.jest.js

```diff
@@ -4,11 +4,6 @@ import NOOP from "lodash/noop";
 import Button from "../Button";
 import { WhatsNew } from "../../Icon/Icons";
 
-jest.mock("../../Icon/Icon", () => {
```

Comment:
after converting button to ts this mock cause errors on running the tests  (about type is undefined) so I thought it can remove in this stage and later we can figure out how to involve mocks in a way that will work with ts